### PR TITLE
[Hotfix] VAR-147 | Fix misalignment on repeat reservation date picker

### PR DESCRIPTION
**Pull request by Rasbin**
This PR should fix misalignment on repeat reservation date picker and minor layout improvement.

### CODE REVIEW

File: app/shared/reservation-confirmation/_confirm-reservation-modal.scss [hunk 1]
```diff
@@ -1,7 +1,7 @@
 .app-ConfirmReservationModal {
   .modal-dialog {
     @media (min-width: $screen-sm-min) {
-      width: 700px;
+      width: 800px;
```
Comment (jimppa) [inline app/shared/reservation-confirmation/_confirm-reservation-modal.scss]: $screen-sm-min is 768px, so you should not create bigger widths than that here. I think creating a new media query for $screen-md-min and setting the new width there would be the correct thing to do.

File: app/shared/date-picker/_date-picker.scss [hunk 2]
```diff
@@ -24,7 +24,8 @@
     position: absolute;
     z-index: 100;
     border: solid 2px $brand-primary;;
-    padding: 20px;
+    padding: 12px;
+    padding-bottom: 0;
```
Comment (jimppa) [inline app/shared/date-picker/_date-picker.scss]: Now you are just overriding 12px padding from the line above, you should combine these paddings (https://developer.mozilla.org/en-US/docs/Web/CSS/padding)

File: app/shared/recurring-reservation-controls/RecurringReservationControls.js [hunk 3]
```diff
@@ -60,10 +60,11 @@ function RecurringReservationControls({
           </Col>
         )}
         {frequency !== '' && (
-          <Col sm={4} xs={12}>
+          <Col sm={5} xs={12}>
             <FormGroup controlId="LastTimeGroup">
               <ControlLabel>{t('RecurringReservationControls.lastTimeLabel')}</ControlLabel>
               <DatePicker
+                className="date-picker"
```
Comment (midori0507) [inline app/shared/recurring-reservation-controls/RecurringReservationControls.js]: Please remove, className `date-picker` already injected in container
Check `app/shared/date-picker/DatePicker.js`

Comment (midori0507) [inline app/shared/recurring-reservation-controls/RecurringReservationControls.js]: Please remove, className date-picker already injected in container
Check app/shared/date-picker/DatePicker.js

File: app/shared/recurring-reservation-controls/_recurring-reservation-controls.scss [hunk 4]
```diff
@@ -6,4 +6,13 @@
     white-space: nowrap;
     overflow: hidden;
   }
+  .date-picker input {
```
Comment (midori0507) [inline app/shared/recurring-reservation-controls/_recurring-reservation-controls.scss]: You are modifying global style of date-picker.

File: app/shared/recurring-reservation-controls/_recurring-reservation-controls.scss [hunk 5]
```diff
@@ -6,4 +6,13 @@
...
+    width: 281px;
```
Comment (midori0507) [inline app/shared/recurring-reservation-controls/_recurring-reservation-controls.scss]: Not the first time i ask you not to put random value..

**Comment by midori0507** [comment]
What you need to do here is make the overlay dropdown shift to right, not left 
Screen Shot 2019-08-13 at 10 20 15

**Pull request merged**